plugins/CoreVisualizations/Visualizations/Graph.php

```diff
@@ -158,31 +163,31 @@ public function afterAllFiltersAreApplied()
 
     }
 
-    protected function addTranslations()
+    protected function addTranslations(): void
     {
         if ($this->config->add_total_row) {
             $totalTranslation = Piwik::translate('General_Total');
-            $this->config->selectable_rows[] = array(
+            $this->selectableRows['total'] = [
```

Comment:
Using the config here instead of the local property actually caused that the totals row was never displayed in the series picker. Not sure if that ever worked, but now it should 🎉